sdk/src/test/java/com/mapbox/maps/MapboxMapTest.kt

```diff
@@ -131,10 +131,11 @@ class MapboxMapTest {
     assertTrue(mapboxMap.isStyleLoadInitiated)
   }
 
+  // TODO fix test
   @Test
-  fun finishLoadingStyleExtension() {
+  fun bindsStyleExtensionComponentsInCorrectOrderAfterStyleDataLoadEvents() {
     val style = mockk<Style>()
-    val styleExtension = mockk<StyleContract.StyleExtension>()
+    val styleExtension = mockk<StyleContract.StyleExtension>(relaxed = true)
```

Comment:
a bit concerned to use `relaxed = true` here - perhaps explicitly mock what we need? or at least use `relaxedUnitFun=true` instead